Zeeguu Reader/Zeeguu Reader/ArticleViewController.swift

```diff
@@ -42,16 +44,23 @@ class ArticleViewController: UIViewController {
 		self.view.backgroundColor = UIColor.whiteColor()
 		if let art = article {
 			let sv = UIScrollView.autoLayoutCapapble()
-			let view = ArticleView(article: art)
-			let views: [String: AnyObject] = ["sv": sv,"v":view, "top":self.topLayoutGuide]
+			articleView = ArticleView(article: art, delegate: self)
+			let views: [String: AnyObject] = ["sv": sv, "v": articleView!]
+			
+			sv.addSubview(refresher)
```

Comment:
i think we should be moving some of this view construction away from the controller to the view.